src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialog.tsx

```diff
@@ -67,6 +68,7 @@ const TransferFundsDialog = ({
       .required()
       .moreThan(0, () => MSG.amountZero),
     tokenAddress: yup.string().address().required(),
+    annotation: yup.string().max(4000),
```

Comment:
I would have renamed this in the component as well (to `annotationMessage`), rather than have to rename it in the transformer, but it doesn't really matter either way

So I'll chuck this to being a personal preference, so you can leave it like this and merge